frontend/src/stories/Introduction.stories.mdx

```diff
@@ -4,12 +4,24 @@ import { Meta } from '@storybook/addon-docs';
 
 # Design System
 
-At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation in Angular or Rails.
+At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
```

Comment:
@psatyal I struggle with the _absoluteness_ of words such as "ensure". The DS cannot ensure it. Only we people developing products can try to do so. The DS is a toolset to make our work easier.

Also, I see the DS not only applied to _user interface_ but also to a general _user experience_. 

```suggestion
At OpenProject, we use a design system to foster that our design delivers a consistent experience to users. The system describes the styles, components and patterns that build the foundation for a consistent user experience. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
```

Reply:
How about "help ensure"? "foster" makes no sense in this sentence I think.